apps/explorer/src/components/transactions/TxCardUtils.tsx

```diff
@@ -44,24 +44,30 @@ export function SuiAmount({
     return <span className={styles.suiamount}>--</span>;
 }
 
-function TxTableHeader({ label }: { label: string }) {
-    return <div className="pl-3">{label}</div>;
+function TxTableHeader({
+    label,
+    smallPaddings,
+}: {
+    label: string;
+    smallPaddings?: boolean;
```

Comment:
It's super weird to me to define this, why does this even have padding baked in to begin with?